profiles/kentik_snmp/raritan/dominion.yml

```diff
@@ -0,0 +1,66 @@
+# https://mibs.observium.org/mib/RemoteKVMDevice-MIB/
+---
+extends:
+  - if-mib.yml
+  - system-mib.yml
+
+sysobjectid:
+  - 1.3.6.1.4.1.13742.3.2.10    # Raritan Dominion SXII DSX2-32
+  - 1.3.6.1.4.1.13742.*    # Raritan Devices
```

Comment:
Generally i find its best to throw the wildcards at the top of the sysOID stack

Reply:
Switched
sysobjectid:
  - 1.3.6.1.4.1.13742.*    # Raritan Devices
  - 1.3.6.1.4.1.13742.3.2.10    # Raritan Dominion SXII DSX2-32